package.json

```diff
@@ -28,9 +28,6 @@
     "email": "ismaelga@gmail.com"
   },
   "main": "dist/react-json-editor.js",
-  "peerDependencies": {
-    "react": ">=0.12.2 <1.0.0"
-  },
```

Comment:
Hi guys! I haven't given the proper attention to this repo since I don't use it anymore and also because there is a great alternative now https://github.com/mozilla-services/react-jsonschema-form

That said. I will try to give attention to the open issues some day soon. And make sure it works with latest React version